model-gen/build/calm/2024-12/core.d.ts

```diff
@@ -0,0 +1,162 @@
+export type Metadata = {
+  [k: string]: unknown;
+}[];
+export type Metadata1 = {
```

Comment:
Should MetaData be a general concept which is leveraged by flow.json and core.json to prevent this